versions.props

```diff
@@ -30,4 +31,6 @@ org.apache.hive:hive-exec = 2.3.8
 org.apache.hive:hive-service = 2.3.8
 org.apache.tez:tez-dag = 0.8.4
 org.apache.tez:tez-mapreduce = 0.8.4
+org.springframework:* = 5.1.10.RELEASE
```

Comment:
why it's not the latest release?

Reply:
Here I think we could align it with the s3mock testing framework, I don't expect to introduce two different versions in apache iceberg repo for testing purpose.